platform/core/src/classes/ImageSet.js

```diff
@@ -1,5 +1,6 @@
 import guid from '../utils/guid.js';
 import OHIFError from './OHIFError';
+import { Vector3 } from 'cornerstone-math';
```

Comment:
Are we still doing this? Do we prefer gl-matrix? Erik mentioned a while back we may try to fade out cornerstone-math?

Reply:
@JamesAPetts this is for you my friend.

Reply:
I'm actually indifferent to what happens here.
We currently don't have `gl-matrix` as an explicit dependency anywhere (other than bundled with `vtk.js` (they aren't peer dependencies there). And we already depend on `cornerstone-math`. Long term I would agree, but I don't think its too much of an issue at the current time.